code/modules/mining/laborcamp/laborshuttle.dm

```diff
@@ -6,10 +6,182 @@
 	possible_destinations = "laborcamp_home;laborcamp_away"
 	req_access = list(ACCESS_BRIG)
 
-
 /obj/machinery/computer/shuttle/labor/one_way
 	name = "prisoner shuttle console"
 	desc = "A one-way shuttle console, used to summon the shuttle to the labor camp."
 	possible_destinations = "laborcamp_away"
 	circuit = /obj/item/circuitboard/labor_shuttle/one_way
-	req_access = list( )
+
+/obj/machinery/computer/shuttle/labor/one_way/allowed(mob/M)
+	. = ..()
+	if(.)
+		return TRUE
```

Comment:
```suggestion
	if(..())
		return TRUE
```
it doesn't matter what `.` is set to in the proc since all return paths of this proc return a preset value